src/Header.spec.js

```diff
@@ -0,0 +1,11 @@
+import { render, screen } from '@testing-library/react'
+import Header from './Header'
+
+describe('render Header', () => {
```

Comment:
```suggestion
describe('Header', () => {
```
rendering the header is what you do in the test but not the subject to be tested.